src/components/ActivityCollector/configValidators.js

```diff
@@ -12,16 +12,14 @@ governing permissions and limitations under the License.
 
 import { string, boolean } from "../../utils/configValidators";
 
-export default () => {
-  return {
-    clickCollectionEnabled: {
-      defaultValue: true,
-      validate: boolean()
-    },
-    downloadLinkQualifier: {
-      defaultValue:
-        "\\.(exe|zip|wav|mp3|mov|mpg|avi|wmv|pdf|doc|docx|xls|xlsx|ppt|pptx)$",
-      validate: string().regexp()
-    }
-  };
```

Comment:
Creating this object inside a function didn't provide any advantage so I simplified it.